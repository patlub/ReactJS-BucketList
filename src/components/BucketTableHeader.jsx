import React, {Component} from 'react';
import '../App.css';

const BucketTableHeader = () => (
    <thead>
    <tr>
        <th>Bucket name</th>
        <th>Description</th>
        <th>Date Added</th>
        <th>Action</th>
    </tr>
    </thead>
);


export default BucketTableHeader;
