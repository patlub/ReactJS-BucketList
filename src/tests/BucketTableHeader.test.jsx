import React from 'react';
import renderer from 'react-test-renderer';
import BucketTableHeader from '../components/BucketTableHeader';


it('Displays bucket table header', () => {
  const rendered = renderer.create(
    <BucketTableHeader />,
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});
