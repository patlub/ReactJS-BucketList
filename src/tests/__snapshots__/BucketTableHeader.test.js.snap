// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays bucket table header 1`] = `
<thead>
  <tr>
    <th>
      Bucket name
    </th>
    <th>
      Description
    </th>
    <th>
      Date Added
    </th>
    <th>
      Action
    </th>
  </tr>
</thead>
`;
